doc/build-instructions/Build_Instructions_V16.md

```diff
@@ -609,4 +650,153 @@ JCL      - d0f36677cbe based on jdk-16+35)
 
 ## AArch64
 
-:construction: Build process for AArch64 (ARMv8 64-bit) Linux is under preparation.
+:penguin:
+The following instructions guide you through the process of building an **OpenJDK V16** binary that contains Eclipse OpenJ9 for AArch64 (ARMv8 64-bit) Linux.
+
+### 1. Prepare your system
+
+The binary can be built on your AArch64 Linux system, or in a Docker container :whale: on x86-64 Linux.
+
+### 2. Get the source
+:penguin:
+First you need to clone the Extensions for OpenJDK for OpenJ9 project. This repository is a git mirror of OpenJDK without the HotSpot JVM, but with an **openj9** branch that contains a few necessary patches. Run the following command:
+```
+git clone https://github.com/ibmruntimes/openj9-openjdk-jdk16.git
+```
+Cloning this repository can take a while because OpenJDK is a large project! When the process is complete, change directory into the cloned repository:
+```
+cd openj9-openjdk-jdk16
+```
+Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Eclipse OMR:
+
+```
+bash get_source.sh
+```
+
+### 3. Prepare for build on AArch64 Linux
+
+You must install a number of software dependencies to create a suitable build environment on your AArch64 Linux system:
+
+- GNU C/C++ compiler (The Docker image uses GCC 7.5)
+- [AArch64 Linux JDK](https://api.adoptopenjdk.net/v3/binary/latest/15/ga/linux/aarch64/jdk/openj9/normal/adoptopenjdk), which is used as the boot JDK.
+- [Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download) - Only when building with `--with-cmake=no`
+
+See [Setting up your build environment without Docker](#setting-up-your-build-environment-without-docker) in [Linux section](#linux) for other dependencies to be installed.
+
+### 4. Create the Docker image
+
+If you build the binary on x86-64 Linux, run the following commands to build a Docker image for AArch64 cross-compilation, called **openj9aarch64**:
+```
+cd openj9/buildenv/docker/aarch64-linux_CC
+docker build -t openj9aarch64 -f Dockerfile .
+```
+
+Start a Docker container from the **openj9aarch64** image with the following command, where `<host_directory>` is the directory that contains `openj9-openjdk-jdk16` in your local system:
+```
+docker run -v /<host_directory>/openj9-openjdk-jdk16:/root/openj9-openjdk-jdk16 -it openj9aarch64
```

Comment:
Please remove the slash before `<host_directory>` for consistency with build instructions for other versions.